### PR TITLE
Added conf for restrict.objectTypes.

### PR DESCRIPTION
**Pull request by GauravTekdi**
Moved the hardcoded list of **restrict.objectTypes** to a variable

### CODE REVIEW

**Comment by GauravTekdi** [comment]
@maheshkumargangula - Can you please review this MR and merge ??

cc : @coolbung

File: kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml [hunk 1]
```diff
@@ -244,6 +244,7 @@ dialcode_external_indexer_parallelism: 1
 dialcode_metric_indexer_parallelism: 1
 schema_definition_cache_expiry_in_sec: 14400
 search_indexer_ignored_fields: ["responseDeclaration", "body", "options", "lastStatusChangedOn", "SYS_INTERNAL_LAST_UPDATED_ON", "sYS_INTERNAL_LAST_UPDATED_ON"]
+search_indexer_restrict_objectTypes: ["EventSet", "EventSetImage", "Event", "EventImage", "Questionnaire", "Misconception", "FrameworkType", "Concept", "Misconception", "Language", "Reference", "Dimension", "Method", "Library", "Domain", "Api"]
```
Comment (coolbung) [inline kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml:247]: Keep all variables lowercase in ansible code.

Comment (GauravTekdi) [reply kubernetes/ansible/roles/flink-jobs-deploy/defaults/main.yml]: Okay, thanks for the response.

**Comment by GauravTekdi** [comment]
@maheshkumargangula - Can you please review this MR and merge ??

cc : @coolbung

**Review by maheshkumargangula — APPROVED** [review]

**Pull request merged**